package.json

```diff
@@ -23,10 +23,13 @@
   "homepage": "https://github.com/actions/http-client#readme",
   "devDependencies": {
     "@types/jest": "^24.0.25",
-    "@types/node": "^13.1.5",
+    "@types/node": "^12.12.24",
     "jest": "^24.9.0",
-    "nock": "^11.7.2",
+    "proxy": "^1.0.1",
     "ts-jest": "^24.3.0",
     "typescript": "^3.7.4"
+  },
+  "dependencies": {
+    "tunnel": "0.0.6"
```

Comment:
@ericsciple I don't see where `tunnel` is used. Should this be moved to `devDependencies` or removed?